examples/machine-set.yaml

```diff
@@ -2,58 +2,40 @@
 apiVersion: machine.openshift.io/v1beta1
 kind: MachineSet
 metadata:
-  name: aws-actuator-testing-machine
-  namespace: default
+  name: local-powervs-machineset
+  namespace: openshift-machine-api
   labels:
-    machine.openshift.io/cluster-api-cluster: aws-actuator-k8s
+    machine.openshift.io/cluster-api-cluster: powervs-k8s-cluster
 spec:
   replicas: 2
   selector:
     matchLabels:
       machine.openshift.io/cluster-api-machineset: test-master
-      machine.openshift.io/cluster-api-cluster: aws-actuator-k8s
+      machine.openshift.io/cluster-api-cluster: powervs-k8s-cluster
   template:
     metadata:
       labels:
         machine.openshift.io/cluster-api-machineset: test-master
-        machine.openshift.io/cluster-api-cluster: aws-actuator-k8s
+        machine.openshift.io/cluster-api-cluster: powervs-k8s-cluster
     spec:
       metadata:
         labels:
           node-role.kubernetes.io/master: ""
           node-role.kubernetes.io/infra: ""
```

Comment:
can we remove this section not to deploy nodes as master and mess the existing cluster?